### PR TITLE
Add Authentiq OAuth2/OIDC provider for HybridAuth v2

### PR DESCRIPTION
**Pull request by ziogaschr**
## Summary

This PR adds support for Authentiq (to HybridAuth v2): A new OAuth2/OIDC provider that offers passwordless authentication and keeps personal data on the phone instead of in the cloud.

* New provider

- [x] Tested

### CODE REVIEW

File: additional-providers/hybridauth-authentiq/Providers/Authentiq.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    if (!$this->user->profile->displayName && $this->user->profile->firstName) {
+        $this->user->profile->displayName = join(' ',
+                                                array(
+                                                  $this->user->profile->firstName,
+                                                  $this->user->profile->lastName
+                                                ));
+    }
+
+    $this->user->profile->email       = @ (property_exists($data, 'email')) ? $data->email : '';
+    $this->user->profile->emailVerified = @ (property_exists($data, 'email_verified')) ? $data->email_verified : '';
```
Comment (ApacheEx) [inline additional-providers/hybridauth-authentiq/Providers/Authentiq.php]: as I see in [response](https://developers.authentiq.com/api-provider/#!/Authentication/userInfo) `email_verified` is boolean, but in HA User object it means `Verified user email` which is string. Here is a [proof](https://github.com/hybridauth/hybridauth/blob/v2/hybridauth/Hybrid/User_Profile.php#L111). 
So, pls use ` $this->user->profile->emailVerified =  $this->user->profile->email;`

File: additional-providers/hybridauth-authentiq/Providers/Authentiq.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  function loginBegin()
+  {
+    // redirect the user to the provider authentication url
+    Hybrid_Auth::redirect( $this->api->authorizeUrl( array( "scope" => $this->scope, "state" => md5( mt_rand() ) ) ) );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  function logout() {
```
Comment (ApacheEx) [inline additional-providers/hybridauth-authentiq/Providers/Authentiq.php]: does this construction make sense?

**Comment by ziogaschr** [comment]
Thanks @ApacheEx will update them

**Comment by ApacheEx** [comment]
once more, thank you!

**Pull request merged**